src/middleware/validator/generic.validator.ts

```diff
@@ -11,12 +11,27 @@ export function validate(req: Request, res: Response, next: NextFunction) {
   next()
 }
 
+export function asInt(idName: string) {
```

Comment:
Maybe change `idName` to `routeParameter`? I'm nitpicking. Idk how I feel about `idName`. 

Ohhhh ya know what else might be nice? Make the parameter have a default value of `id`. Doing so would allow us to not have to pass the string 'id' for 80% of the invocations of this function. 

Looks like
```suggestion
export function asInt(route parameter = 'id') {
```

Reply:
Might want to throw some docs on this function since it's kinda weird (not really weird, more just different) that it's wrapping a middleware. 

Also you could give an example on how to call this will multiple ids using something like

```
Route.get('/something/:I'd/other thing/:otherid', ['id', 'otherid'].map(id => asInt(id)), ...)
```

Reply:
Or or, we could make this function accept a string or an array of strings, so as to clean up the routers a bit. Bit tbh that feels like a bad idea since it'll make this code a bit tougher to read through.

Reply:
Updated in commit